.github/workflows/docker_publish.yml

```diff
@@ -253,121 +82,52 @@ jobs:
         moab_versions : [
          5.3.0,
         ]
-    container: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-${{ matrix.compiler}}-ext-hdf5_${{ matrix.hdf5_versions}}-moab_${{ matrix.moab_versions }}:ci_testing
-
-    name: Installing and Testing DAGMC
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-        with:
-          submodules: recursive
-
-      - name: Setup
-        run: |
-          git config --global --add safe.directory ${GITHUB_WORKSPACE}
-          git config --global --add safe.directory /github/home/
-          git config --global --add safe.directory /github/home/.cache
-          git config --global --add safe.directory /github/home/.cache/pip
-          git config --global --add safe.directory /github/home/.cache/pip/http
-          echo "MOAB_VERSION=${{ matrix.moab_versions }}" >> $GITHUB_ENV
-          echo "COMPILER=${{ matrix.compiler }}" >> $GITHUB_ENV
-          echo "HDF5_VERSION=${{ matrix.hdf5_versions }}" >> $GITHUB_ENV
-          echo "REPO_SLUG=${GITHUB_REPOSITORY}" >> $GITHUB_ENV
-          echo "PULL_REQUEST=$(echo  $GITHUB_REF | cut -d"/" -f3)" >> $GITHUB_ENV
-          echo "DOUBLE_DOWN="OFF"" >> $GITHUB_ENV
-          ln -s $GITHUB_WORKSPACE /root/build_dir/DAGMC
-
-      - name: Install DAGMC
-        run: |
-          cd $GITHUB_WORKSPACE
-          /bin/bash "CI/scripts/install.sh"
-
-      - name: Run DAGMC Test
-        run: |
-          cd $GITHUB_WORKSPACE
-          /bin/bash "CI/scripts/tests.sh"
-
-  pushing_housekeeping_stable_img:
-    if: ${{ github.repository_owner == 'svalinn' }}
-    needs: [test_img]
-    runs-on: ubuntu-latest
-    env:
-      hdf5_versions: 1.10.4
-      hdf5_build_dir: hdf5_build_dir
-    strategy:
-      matrix:
-        ubuntu_versions : [18.04,]
-
-    name: Housekeeping':' latest -> stable
-    steps:
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
+        static_exe : [
+         OFF,
+         ON,
+        ]
```

Comment:
unused build-arg

Reply:
resolved by removing these lines from the matrix